src/main/java/com/google/sps/servlets/DatabaseLoaderServlet.java

```diff
@@ -30,6 +33,7 @@
 public class DatabaseLoaderServlet extends HttpServlet {
 
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private final LatLng London = new LatLng(51.509865, -0.1278);
```

Comment:
If this is a constant, then we like to use CAPITALS for their names: https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names

```suggestion
  private final LatLng LONDON = new LatLng(51.509865, -0.1278);
```

Reply:
Done.